README.md

```diff
@@ -1,39 +1,66 @@
-[![Stories in Ready](https://badge.waffle.io/thinkthroughmath/dev_wizard.png?label=ready&title=Ready)](https://waffle.io/thinkthroughmath/dev_wizard)
 # DevWizard
 
+## Issue and Feature Tracking
+
+[![Stories in Ready](https://badge.waffle.io/thinkthroughmath/dev_wizard.png?label=ready&title=Ready)](https://waffle.io/thinkthroughmath/dev_wizard)
+
 ## Setup
 
-Install Elixir: `$ brew install elixir`
+Instructions are written for setup and usage on Mac OSX, but should work on
+other supported platforms.
 
 ## Run Locally
 
-To start your Phoenix app:
+### Requirements/Dependencies
 
-  1. Install dependencies with `mix deps.get`
-  2. Create and migrate your database with `mix ecto.create && mix ecto.migrate`
-  3. Create a file called `secret_vars.sh`. See section on secret_vars.sh.
-  4. Start Phoenix endpoint with `./run_development.sh`
+  1. Elixir: http://elixir-lang.org/
+  2. Node.js: https://nodejs.org/en/
+  3. Postgres: http://www.postgresql.org/
 
-Now you can visit [`localhost:4000`](http://localhost:4000) from your browser.
+### Setup
+
+  1. Install Elixir: `$ brew install elixir`
+  2. Install dependencies: `$ mix deps.get`
+  3. Create and migrate your database: `$ mix ecto.create && mix ecto.migrate`
+  4. Create a file called `secret_vars.sh`. See section on `secret_vars.sh` for details.
 
-### secret_vars.sh
+  Note: You may have to run `$ npm install` when prompted.
 
-This file contains sensitive keys and should not be committed. Create your own
-with GitHub application keys.
+#### secret_vars.sh
 
-The file should look like this:
+This file contains sensitive keys and should not be committed to source control.
+This file is required to run the DevWizard. Create your own version with your
+GitHub application keys.
+
+The file should contain the following lines:
 
 ```
 export GH_CLIENT_ID=<YOUR GITHUB CLIENT ID>
 export GH_CLIENT_SECRET=<YOUR GITHUB CLIENT SECRET>
 ```
 
+Note: You may need to setup these keys through GitHub.
+For details, see: https://github.com/blog/1509-personal-api-tokens
```

Comment:
This isn't actually personal api tokens, these are created from here: `https://github.com/settings/applications/new`